freyja_adapters/digital_twin/ibeji_adapter/README.md

```diff
@@ -0,0 +1,31 @@
+# Ibeji Adapter
+
+The Ibeji Adapter is used to integrate with the [Ibeji In-Vehicle Digital Twin Service](https://github.com/eclipse-ibeji/ibeji), and optionally [Chariott](https://github.com/eclipse-chariott/chariott) to discover Ibeji.
+
+## Configuration
+
+The `res` directory contains two sample config files. The `ibeji_adapter_config.sample.json` will be copied to the build output, so you can edit this file to configure the adapter.
+
+### Common Configuration Fields
+
+Whether or not you are using Chariott, the following config values are required:
+
+- `service_type`: Specifies whether to call Ibeji directly or use Chariott's Service Discovery system to find it. Valid values are `"ChariottDiscoveryService"` and `"InVehicleDigitalTwinService"`.
```

Comment:
I know that you are just copying this, so no need to change it, but you may want to rethink the name of this in the future.

I see that there are two options for locating the In-Vehicle Digital Twin, statically via an uri in the config or dynamically at runtime via Chariott's discovery service. service_type does not seem an appropriate term to use to represent these two options.

Reply:
I agree. I will address this in #4 which will improve the config for the adapters here in the same way as I've been doing in the Freyja repo. Perhaps something like `service_discovery_mode`?

Reply:
Yes, much better.